src/event/special_user_message.rs

```diff
@@ -358,6 +372,36 @@ fn deploy_command<'a>(
             .send()
             .await
             .map_err(|e| format!("API Error: {:?}", e))?;
+        let res_json = res
+            .json::<BuildsResponse>()
+            .await
+            .map_err(|_| "Invalid API Response")?;
```

Comment:
I think we should do a better job of preserving this error, maybe doing `format!("Invalid API Response: {}", e)` if we're going to be returning a String.